2-mandatory/1-weather-report.js

```diff
@@ -11,25 +11,29 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
-}
-
+    function getTemperatureReport(cities){
```

Comment:
This looks good 👍 
One small suggestion - keep an eye on indentation, as it will make it easier for other developers to read your code.